Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.py

```diff
@@ -1196,6 +1207,86 @@ def file_replace_existing_command(client: 'GSuiteClient', args: dict[str, str])
     return handle_response_file_single(file, args)
 
 
+def modify_label_command(client: 'GSuiteClient', args: dict[str, str]) -> CommandResults:
+    modify_label_request_res = prepare_file_modify_labels_request(
+        client, args, scopes=COMMAND_SCOPES['MODIFY_LABELS_PERMISSIONS_CRUD'])
+    http_request_params = modify_label_request_res['http_request_params']
+
+    url_suffix = URL_SUFFIX['FILE_MODIFY_LABEL'].format(args.get('file_id'))
+    body_request = {
+        "kind": "drive#modifyLabelsRequest",
+        "labelModifications": [
+            {
+                "fieldModifications": [
+                    {
+                        "kind": "drive#labelFieldModification",
+                        "fieldId": args.get('field_id'),
+                        "setSelectionValues": [
+                            args.get('selection_label_id')
+                        ]
+                    }
+                ],
+                "kind": "drive#labelModification",
+                "labelId": args.get('label_id'),
+                "removeLabel": args.get('remove_label', False)
+            }
+        ]
+    }
+
+    response = client.http_request(url_suffix=url_suffix, method='POST', params=http_request_params, body=body_request)
+
+    table_hr_md = tableToMarkdown(HR_MESSAGES['MODIFY_LABEL_SUCCESS'].format(args.get('file_id')),
+                                  response,
+                                  headerTransform=pascalToSpace,
+                                  removeNull=False)
+
+    return CommandResults(
+        outputs=response,
+        readable_output=table_hr_md,
+    )
+
+
+def get_file_labels_command(client: 'GSuiteClient', args: dict[str, str]) -> CommandResults:
+    modify_label_request_res = prepare_file_modify_labels_request(
+        client, args, scopes=COMMAND_SCOPES['MODIFY_LABELS_PERMISSIONS_CRUD'])
+    http_request_params = modify_label_request_res['http_request_params']
+
+    url_suffix = URL_SUFFIX['FILE_GET_LABELS'].format(args.get('file_id'))
+
+    response = client.http_request(url_suffix=url_suffix, method='GET', params=http_request_params)
+
+    table_hr_md = tableToMarkdown(HR_MESSAGES['GET_LABEL_SUCCESS'].format(args.get('file_id')),
+                                  response,
+                                  headerTransform=pascalToSpace,
+                                  removeNull=False)
+
+    return CommandResults(
+        outputs=response,
+        readable_output=table_hr_md,
+    )
+
+
+def get_labels_command(client: 'GSuiteClient', args: dict[str, str]) -> CommandResults:
+    modify_label_request_res = prepare_get_labels_request(
+        client, args, scopes=COMMAND_SCOPES['MODIFY_LABELS_PERMISSIONS_CRUD'])
+    http_request_params = modify_label_request_res['http_request_params']
+
+    full_url = URLS['DRIVE_LABELS'] + '?' + urllib.parse.urlencode(http_request_params)
+    demisto.log(full_url)
```

Comment:
We don't really use demisto.log anymore let's change to demisto.info and add something more indicative
```suggestion
    demisto.info(f'full url for get labels is: {full_url}')
```